src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/probation/ProbationEventListenerIntTest.kt

```diff
@@ -148,23 +148,25 @@ class ProbationEventListenerIntTest : MessagingMultiNodeTestBase() {
     }
 
     @Test
-    fun `should link person to an existing NOMIS record`() {
+    fun `should link new probation record to an existing prison record`() {
+      telemetryRepository.deleteAll()
```

Comment:
Unfortunately I don't think there is a way around this as the telemetry does not contain any identifying features - UUID for example